godot-kotlin/godot-library/src/nativeCore/kotlin/godot/core/type/Basis.kt

```diff
@@ -762,6 +795,72 @@ class Basis(
         result = 31 * result + _z.hashCode()
         return result
     }
+
+
+    fun set(xAxis: Vector3, yAxis: Vector3, zAxis: Vector3) {
+        setAxis(0, xAxis)
+        setAxis(1, yAxis)
+        setAxis(2, zAxis)
+    }
+
+    /*
+     * GDScript related members
```

Comment:
Everything below here are only available in GDScript, i.e meaning the core cpp implementation does not have them declared.